src/Components/SmartComponents/CVEs/CVEsTable.js

```diff
@@ -52,9 +51,9 @@ const CVEsTableWithContext = ({ context, header, entity }) => {
 
     const rows = cves.data
     .map(cve => (selectedCves.includes(cve.id) && { ...cve, selected: true }) || cve)
-    .map(cve => {
-        const current = openedCves.find(opened => opened === cve.id);
-        return ({ ...cve, isOpen: current && true || false });
+    .map((cve, index) => {
+        const current = index % 2 === 0 ? openedCves.includes(cve.id) : undefined;
```

Comment:
How about using `index % 2 === 0 && openedCves.includes(cve.id)`;

Reply:
Same deal, we need undefined here not false.